### ISSUE TITLE
Unexpected error occurred during Jinja2 environment setup: unable to locate collection middleware_automation.amq

### ISSUE THREAD

**Issue by RobertFloor** [ansible-middleware/amq#32]
##### SUMMARY
Running the default playbook and getting a problem. 

##### ISSUE TYPE
 - Bug Report
 Trying to run the latest version of the code gives me the following error:
```
ansible-galaxy collection install -r requirements.yml --force
ansible-playbook -i hosts_vagrant.yml playbooks/activemq.yml -v
```
 ```
 _______________________________________________________
< TASK [activemq : Parse passwd for existing user salt] >
 -------------------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

[WARNING]: an unexpected error occurred during Jinja2 environment setup: unable to locate collection middleware_automation.amq
fatal: [192.168.2.211]: FAILED! => {"msg": "template error while templating string: unable to locate collection middleware_automation.amq. String: {{ item.password | middleware_automation.amq.pbkdf2_hmac(hexsalt=existing_user[0]) }}"}
[WARNING]: an unexpected error occurred during Jinja2 environment setup: unable to locate collection middleware_automation.amq
fatal: [192.168.2.212]: FAILED! => {"msg": "template error while templating string: unable to locate collection middleware_automation.amq. String: {{ item.password | middleware_automation.amq.pbkdf2_hmac(hexsalt=existing_user[0]) }}"}
```
##### ANSIBLE VERSION

```

```

##### COLLECTION VERSION

```

```

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```

```

**Comment by guidograzioli** [comment]
Hello Robert, to have ansible-playbook find the filter it is not enough to clone and run form there, the collection needs to be "properly" installed (with galaxy collection install, or just drop it or symlink it to ~/.ansible/collections/ansible_collections/ )

**Comment by RobertFloor** [comment]
Hi Guido thanks for the help it indeed works. However, the Readme says to run this command but that does not work: 
```
❯ ansible-galaxy collection install middleware_automation-amq
ERROR! Neither the collection requirement entry key 'name', nor 'source' point to a concrete resolvable collection artifact. Also 'name' is not an FQCN. A valid collection name must be in the format <namespace>.<collection>. Please make sure that the namespace and the collection name contain characters from [a-zA-Z0-9_] only.

Could not find middleware_automation-amq.
```

The second approach from the readme did work:

```
 ansible-galaxy collection build .
 ansible-galaxy collection install middleware_automation-amq
```

**Comment by guidograzioli** [comment]
yup good catch!

**Comment by guidograzioli** [comment]
Fixed in 1.1.1 bump commit
